spec/typescript-spec.coffee

```diff
@@ -0,0 +1,36 @@
+typescript = require '../src/typescript'
+crypto = require 'crypto'
+
+describe "TypeScript transpiler support", ->
+  beforeEach ->
+    jasmine.snapshotDeprecations()
```

Comment:
I don't think these should be needed since the TypeScript addition shouldn't have any deprecations to start with, unless I'm missing something.